infrastructure/templates/template.json

```diff
@@ -997,16 +997,54 @@
         "criterionType": "StaticThresholdCriterion"
       },
       {
-        "name": "[concat(parameters('subscription'),'DataFactoryPipelineFailures')]",
-        "description": "Data Factory - Pipeline Failed Run",
-        "resourceId": "[resourceId('Microsoft.DataFactory/factories', variables('dataFactoryName'))]",
-        "resourceType": "Microsoft.DataFactory/factories",
-        "metricName": "PipelineFailedRuns",
-        "threshold": 1,
-        "operator": "GreaterThan",
-        "timeAggregation": "Total",
-        "criterionType": "StaticThresholdCriterion"
-      }
+        "type": "microsoft.insights/metricAlerts",
+        "apiVersion": "2018-03-01",
+        "name": "[parameters('metricAlerts_Data_Factory_failed_pipeline_runs_name')]",
+        "location": "global",
+        "dependsOn": [
+            "[resourceId('microsoft.insights/actionGroups', parameters('actionGroups_s101p01_ag_ees_alertedusers_name'))]"
+        ],
+        "tags": {
+            "Environment": "Prod",
+            "Parent Business": "Business Intelligence",
+            "Portfolio": "Data Directorate",
+            "Product": "Enterprise Data and Analytics Platform (EDAP)",
+            "Service Line": "Data Operations",
+            "Service Offering": "Enterprise Data and Analytics Platform (EDAP)",
+            "Service": "Business Intelligence"
+        },
+        "properties": {
+            "severity": 1,
+            "enabled": true,
+            "scopes": [
+                "[parameters('factories_s101p01_df_ees_release_externalid')]"
+            ],
+            "evaluationFrequency": "PT1M",
+            "windowSize": "PT5M",
+            "criteria": {
+                "allOf": [
+                    {
+                        "threshold": 1,
+                        "name": "Metric1",
+                        "metricNamespace": "Microsoft.DataFactory/factories",
+                        "metricName": "PipelineFailedRuns",
+                        "operator": "GreaterThan",
+                        "timeAggregation": "Total",
+                        "criterionType": "StaticThresholdCriterion"
+                    }
+                ],
+                "odata.type": "Microsoft.Azure.Monitor.SingleResourceMultipleMetricCriteria"
+            },
+            "autoMitigate": true,
+            "targetResourceType": "Microsoft.DataFactory/factories",
+            "actions": [
+                {
+                    "actionGroupId": "[resourceId('microsoft.insights/actionGroups', parameters('actionGroups_s101p01_ag_ees_alertedusers_name'))]",
```

Comment:
The parameter `actionGroups_s101p01_ag_ees_alertedusers_name` doesn't exist in our template - after exporting a template in Azure Portal, you need to substitute all the variables/parameters it declares from ones we have in our template. 

In the instance of the actionGroupId, you can get that from the existing "microsoft.insights/metricAlerts" - line 4027ish:

```
        "actions": [
          {
            "actionGroupId": "[resourceId('microsoft.insights/actionGroups', variables('actionGroupAlerts'))]"
          }
```

Similar changes will be need to made to other parameters referenced.